src/reducers/globalReducer.js

```diff
@@ -0,0 +1,31 @@
+import initialStore from '../store/initialState';
+import {
+  TRIGGER_AUTHENTICATED,
+  TRIGGER_ERROR,
+} from '../actions/actionTypes';
+
+const { global } = initialStore;
```

Comment:
#### Nitpick 
- Why the choice of name `globalReducer` for this file?

Reply:
It is generic to the `global` state declared in the app store.

*initialStore.js*
<img width="518" alt="Screen Shot 2019-04-10 at 9 28 12 AM" src="https://user-images.githubusercontent.com/30989030/55863780-21ec1f00-5b73-11e9-9c00-4e7f07fcc7bd.png">

Reply:
@devdbrandy , What is this `global` state? You know `global` is a reserved word in lots of coding languages right?

Reply:
Yes, I'm aware. However, I have made changes to the initial state.